app/vmauth/auth_config.go

```diff
@@ -31,7 +32,8 @@ var (
 
 // AuthConfig represents auth config.
 type AuthConfig struct {
-	Users []UserInfo `yaml:"users,omitempty"`
+	Users     []UserInfo         `yaml:"users,omitempty"`
+	IPFilters *ipfilters.IPLists `yaml:"ip_filters,omitempty"`
```

Comment:
It would be better to do not add `IPFilters` field in the open-source version of vmauth, since it isn't supported there. This field may confuse users of open-source vmauth.

Reply:
fixed